src/citrine/resources/ara_definition.py

```diff
@@ -161,3 +169,29 @@ def preview(self, defn: AraDefinition, preview_roots: List[LinkByUID]) -> dict:
             "rows": [x.as_dict() for x in preview_roots]
         }
         return self.session.post_resource(path, body)
+
+    def register(self, defn: AraDefinition) -> AraDefinition:
+        """
+        [ALPHA] Register an Ara Definition.
+
+        If the provided AraDefinition does not have a definition_uid, create a new element of the
```

Comment:
@alexandraj777 here is where we're using the same `register` method for create and for update. This follows a pattern that's used elsewhere in citrine-python.  Update increments the version number and assigns a new version_uid (server-side).

Reply:
So, what's the serverside call? Is it the case that clientside create and update both call the same serverside register, or that clientside create calls serverside create, and clientside update calls serverside update, and serverside create and update call the same register? 

I ask because the former couples clientside architecture decisions tightly to serverside architecture decisions, and the latter allows flexibility for the server to change the implementation as needed.